packages/vm/src/cache.rs

```diff
@@ -389,7 +389,7 @@ mod tests {
     static IBC_CONTRACT: &[u8] = include_bytes!("../testdata/ibc_reflect.wasm");
 
     fn default_features() -> HashSet<String> {
-        features_from_csv("iterator,staking")
+        features_from_csv("abort,iterator,staking")
```

Comment:
We can remove `abort` from all these features lists if you make the changes requested above.